Makefile tweaks

ynezz opened the pull request:
Hi,

this small addition allows me to use this options in Makefile.config:

CROSS_COMPILE = /opt/devel/nxp/toolchain/gcc-arm-none-eabi-4_7-2013q1/bin/arm-none-eabi-
JLINK_EXE = /opt/devel/jtag/segger-jlink/JLink_Linux_V485d_x86_64/StartJLinkExe.sh

As I usually prefer to explicitly select tools of my choice. For the JLinkExe commandline option, nrftool [Add jlinkexe commandline option](567ef6d43630202b5d1d282e0da76dc911ea987e) addition is needed.

Thanks.

ynezz committed:
Allow custom specification of the toolchain path

Signed-off-by: Petr Štetiar <ynezz@true.cz>

ynezz committed:
Fix Makefile.config include path in example's Makefile

Signed-off-by: Petr Štetiar <ynezz@true.cz>

ynezz committed:
Allow specification of the path to JlinkExe

This needs updated version of nrftool which add --jlinkexe option.

Signed-off-by: Petr Štetiar <ynezz@true.cz>

pauloborges commented:
This looks great! but before merging this PR we need to:
1. Modify [nrftool](https://github.com/pauloborges/nrftool) to add `--jlink` parameter
2. Rebase\* your `maketweaks` branch on top of my modified `devel` branch.

BTW, your link "[Add jlinkexe commandline option](https://github.com/pauloborges/blessed/pull/567ef6d43630202b5d1d282e0da76dc911ea987e)" is broken.

*As you can see, I've pushed some commits onto `devel` using `--force`. Soon I will keep it stable and use feature branches (I'm just waiting for cktakahasi to review those commits) but I can't guarantee that it will not change again this week.

ynezz committed:
Allow specification of the programmer command

This change should allow easy change to for example 'nrftool --verbose' in my
custom Makefile.config.

Signed-off-by: Petr Štetiar <ynezz@true.cz>

ynezz commented:
I've issued separate PR for the nrftool change and rebased my changes on top of your latest devel branch. Thanks.

pauloborges commented:
Thanks!
